src/index.js

```diff
@@ -0,0 +1,52 @@
+import { from } from 'form-data'; // eslint-disable-line
+import './style.css'; // eslint-disable-line
+import { ToDoList } from './arr-crud'; // eslint-disable-line
+import { Task } from './arr-crud'; // eslint-disable-line
+import { dragndrop, renderTasks } from './drag-and-drop.js'; // eslint-disable-line
+
+export const toDoListArr = new ToDoList();
+
+const addToDo = document.querySelector('.add-item');
+const clearAllCompleted = document.querySelector('.btn-clear-all');
+
+function setStorage(ToDoArr) {
+  localStorage.setItem('ToDo', JSON.stringify(ToDoArr));
+}
```

Comment:
Following single principle and modular patterns, storage functions should not be spread across unrelated code, I would suggest you create a class to handle all local storage functions.